dirvish.el

```diff
@@ -1228,7 +1229,8 @@ default implementation is `find-args' with simple formatting."
 (defun dirvish-update-body-h ()
   "Update UI of current Dirvish."
   (when-let ((dv (dirvish-curr)))
-    (cond ((eobp) (forward-line -1)) ((bobp) (forward-line 1)))
+    (cond ((eobp) (forward-line -1))
+          ((bobp) (while (invisible-p (point)) (forward-line 1))))
```

Comment:
> `(forward-line 2)` would be another solution but I am not sure if everyone has the same 2 lines at the beginning

I believe it can be written as follows

```emacs-lisp
(cond ((eobp) (forward-line -1))
      ((bobp) (forward-line (if dirvish--dired-free-space 2 1))))
```

Reply:
> I believe it can be written as follows
> 
> ```emacs-lisp
> (cond ((eobp) (forward-line -1))
>       ((bobp) (forward-line (if dirvish--dired-free-space 2 1))))
> ```

Thanks, I changed it to this.